releases/mainnet/1.9.6/contracts.json

```diff
@@ -0,0 +1,260 @@
+  "MCD_CLIP_UNIV2DAIUSDT_A": "0xe4B82Be84391b9e7c56a1fC821f47569B364dd4a",
+  "MCD_CLIP_CALC_UNIV2DAIUSDT_A": "0x4E88cE740F6bEa31C2b14134F6C5eB2a63104fcF",
+  "MIP21_LIQUIDATION_ORACLE": "0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30",
+  "RWA001": "0x10b2aA5D77Aa6484886d8e244f0686aB319a270d",
+  "PIP_RWA001": "0x76A9f30B45F4ebFD60Ce8a1c6e963b1605f7cB6d",
+  "MCD_JOIN_RWA001_A": "0x476b81c12Dc71EDfad1F64B9E07CaA60F4b156E2",
+  "RWA001_A_URN": "0xa3342059BcDcFA57a13b12a35eD4BBE59B873005",
+  "RWA001_A_INPUT_CONDUIT": "0x486C85e2bb9801d14f6A8fdb78F5108a0fd932f2",
+  "RWA001_A_OUTPUT_CONDUIT": "0xb3eFb912e1cbC0B26FC17388Dd433Cecd2206C3d",
+  "RWA002": "0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23",
+  "PIP_RWA002": "0xd2473237E20Bd52F8E7cE0FD79403A6a82fbAEC8",
+  "MCD_JOIN_RWA002_A": "0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152",
+  "RWA002_A_URN": "0x225B3da5BE762Ee52B182157E67BeA0b31968163",
+  "RWA002_A_INPUT_CONDUIT": "0x2474F297214E5d96Ba4C81986A9F0e5C260f445D",
+  "RWA002_A_OUTPUT_CONDUIT": "0x2474F297214E5d96Ba4C81986A9F0e5C260f445D",
+  "RWA003": "0x07F0A80aD7AeB7BfB7f139EA71B3C8f7E17156B9",
+  "PIP_RWA003": "0xDeF7E88447F7D129420FC881B2a854ABB52B73B8",
+  "MCD_JOIN_RWA003_A": "0x1Fe789BBac5b141bdD795A3Bc5E12Af29dDB4b86",
+  "RWA003_A_URN": "0x7bF825718e7C388c3be16CFe9982539A7455540F",
+  "RWA003_A_INPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
+  "RWA003_A_OUTPUT_CONDUIT": "0x2A9798c6F165B6D60Cfb923Fe5BFD6f338695D9B",
+  "RWA004": "0x873F2101047A62F84456E3B2B13df2287925D3F9",
+  "PIP_RWA004": "0x5eEE1F3d14850332A75324514CcbD2DBC8Bbc566",
+  "MCD_JOIN_RWA004_A": "0xD50a8e9369140539D1c2D113c4dC1e659c6242eB",
+  "RWA004_A_URN": "0xeF1699548717aa4Cf47aD738316280b56814C821",
+  "RWA004_A_INPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
+  "RWA004_A_OUTPUT_CONDUIT": "0xe1ed3F588A98bF8a3744f4BF74Fd8540e81AdE3f",
+  "RWA005": "0x6DB236515E90fC831D146f5829407746EDdc5296",
+  "PIP_RWA005": "0x8E6039C558738eb136833aB50271ae065c700d2B",
+  "MCD_JOIN_RWA005_A": "0xA4fD373b93aD8e054970A3d6cd4Fd4C31D08192e",
+  "RWA005_A_URN": "0xc40907545C57dB30F01a1c2acB242C7c7ACB2B90",
+  "RWA005_A_INPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
+  "RWA005_A_OUTPUT_CONDUIT": "0x5b702e1fEF3F556cbe219eE697D7f170A236cc66",
+  "RWA006": "0x4EE03cfBF6E784c462839f5954d60f7C2B60b113",
+  "PIP_RWA006": "0xB8AeCF04Fdf22Ef6C0c6b6536896e1F2870C41D3",
+  "MCD_JOIN_RWA006_A": "0x5E11E34b6745FeBa9449Ae53c185413d6EdC66BE",
+  "RWA006_A_URN": "0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb",
+  "RWA006_A_INPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
+  "RWA006_A_OUTPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
+  "PROXY_PAUSE_ACTIONS": "0x6bda13D43B7EDd6CAfE1f70fB98b5d40f61A1370",
+  "PROXY_DEPLOYER": "0x1b93556AB8dcCEF01Cd7823C617a6d340f53Fb58",
+  "MCD_FLASH": "0x1EB4CF3A948E7D72A198fe073cCb8C7a948cD853",
+  "VOTE_DELEGATE_PROXY_FACTORY": "0xD897F108670903D1d6070fcf818f9db3615AF272",
```

Comment:
I think these two are missing in Goerli, can be them added as well? The contracts should be deployed.

Reply:
done